website/docs/r/search_service.html.markdown

```diff
@@ -108,6 +108,10 @@ The following arguments are supported:
 
 * `replica_count` - (Optional) Specifies the number of Replica's which should be created for this Search Service. This field cannot be set when using a `free` sku ([see the Microsoft documentation](https://learn.microsoft.com/azure/search/search-sku-tier)).
 
+* `semantic_search_sku` - (Optional) The Semantic Search SKU which should be used for this Search Service. Possible values include `disabled`, `free` and `standard`. Defaults to `disabled`.
```

Comment:
Updated to remove the `disabled` value, but typically these are prefixed with `Specifies`:

```suggestion
* `semantic_search_sku` - (Optional) Specifies the Semantic Search SKU which should be used for this Search Service. Possible values include `free` and `standard`.
```

Reply:
Fixed.